Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'hacker_news_search', github: 'cdmwebs/hacker_news_search'
```

Comment:
I don't think this won't work for the gem release. Can we get your changes pushed upstream?